src/components/forms/SelectField/SelectField.js

```diff
@@ -0,0 +1,69 @@
+import {FormFieldDescription, FormLabel, Paragraph} from '@utrecht/component-library-react';
+import {Field} from 'formik';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+import {getBEMClassName} from '../../../utils';
+
+export const SelectField = ({
+  name = '',
+  label = '',
+  id = '',
+  isRequired = false,
+  description = '',
+  disabled = false,
+  invalid = false,
+  options = [],
+}) => {
+  const labelClassName = getBEMClassName('label', [isRequired && 'required'].filter(Boolean));
+
+  return (
+    <>
+      <Paragraph className={labelClassName}>
+        <FormLabel htmlFor={id}>{label}</FormLabel>
+      </Paragraph>
+      <Field
+        name={name}
+        component={Select}
+        inputId={id}
+        components={{DropdownIndicator: () => null, IndicatorSeparator: () => null}}
+        classNames={{
+          control: state => {
+            return getBEMClassName('rs-select', [
+              state.isFocused && 'focus',
+              disabled && 'disabled',
+              invalid && 'invalid',
+              isRequired && 'required',
+            ]);
+          },
+          menu: () => {
+            return getBEMClassName('rs-select-menu');
+          },
+          option: state => {
+            return getBEMClassName('rs-select-option', [state.isFocused && 'focus']);
+          },
+        }}
+        options={options}
+        unstyled
+        isDisabled={disabled}
+      />
+      {description && <FormFieldDescription invalid={invalid}>{description}</FormFieldDescription>}
+    </>
+  );
+};
+
+SelectField.propTypes = {
+  name: PropTypes.string.isRequired,
+  label: PropTypes.string,
+  id: PropTypes.string,
+  isRequired: PropTypes.bool,
+  description: PropTypes.string,
+  disabled: PropTypes.bool,
+  invalid: PropTypes.bool,
+  options: PropTypes.arrayOf(
+    PropTypes.shape({
+      label: PropTypes.string,
+      value: PropTypes.string,
+    })
+  ).isRequired,
```

Comment:
this is a bit too constrained, see https://github.com/open-formulieren/formio-builder/blob/main/src/components/formio/select.tsx#LL40C3-L40C16

-> you can pass the `getOptionValue` prop to react-select so that it supports any option shape, really. I think you can only really demand that `options: PropTypes.array,` and there isn't much more to say about that.